script/tool/lib/src/analyze_command.dart

```diff
@@ -62,15 +68,20 @@ class AnalyzeCommand extends PluginCommand {
         await processRunner.runAndStream('flutter', <String>['packages', 'get'],
             workingDir: package, exitOnError: true);
       } else {
-        await processRunner.runAndStream('pub', <String>['get'],
+        await processRunner.runAndStream('dart', <String>['pub', 'get'],
             workingDir: package, exitOnError: true);
       }
     }
 
+    // Use the Dart SDK override if one was passed in.
+    final String dartSdk = argResults![_dartSdk] as String;
+    final String dartBinary =
+        dartSdk == null ? ' dart' : p.join(dartSdk, 'bin/dart');
```

Comment:
s/'bin/dart'/'bin', 'dart'/

Ideally this tooling should work on Windows; I'm sure there are lots of issues at the moment, but I'd rather not add new ones when it's easy not to.